Geometry/CMSCommonData/python/cmsExtendedGeometry2023XML_cfi.py

```diff
@@ -233,7 +234,7 @@
         'Geometry/MuonCommonData/data/v7/gem21.xml',
         'Geometry/MuonCommonData/data/v2/csc.xml',
         'Geometry/MuonCommonData/data/PhaseII/mfshield.xml',
-        'Geometry/MuonCommonData/data/v8/me0.xml',
+        'Geometry/MuonCommonData/data/PhaseII/me0.xml',
```

Comment:
@lgray and @bsunanda - please, agree which version of me0 should be used. We should not be flipping it back and forth.